client/my-sites/plans-features-main/index.jsx

```diff
@@ -607,7 +608,15 @@ export class PlansFeaturesMain extends Component {
 				<QuerySites siteId={ siteId } />
 				<QuerySitePlans siteId={ siteId } />
 				<HappychatConnection />
-				<div className="plans-features-main__notice" />
+				<PlanNotice
+					visiblePlanNames={ visiblePlans }
+					siteId={ siteId }
+					isInSignup={ this.props.isInSignup }
+					discountInformation={ {
+						withDiscount: this.props.withDiscount,
+						discountEndDate: this.props.discountEndDate,
+					} }
+				/>
```

Comment:
Since the notice needs to be displayed only on `/plans`, and it never is displayed in signup flows, would it be possible to render the `<PlanNotice/>` component one level above, maybe here?
https://github.com/Automattic/wp-calypso/blob/1d90b031e834b9664a7e8b45d762ee421af3930d/client/my-sites/plans/main.jsx#L267

I realize that the `visiblePlans` prop won't be available there, but perhaps we can get it via some selector? This would truly separate the notice from the plans grid. 🤔